### DISCUSSION TITLE
Conheça a nova ferramenta para renovar a sua Associação na APyB

### DISCUSSION THREAD

**Discussion by tiidadavena** [apyb/comunidade#139] [Divulgação]
## Conheça a nova ferramenta para renovar a sua Associação na APyB

Buscando evoluir as ferramentas a partir das demandas e sugestões da comunidade, anunciamos que estamos migrando a nossa Plataforma de Associação de forma a centralizar todos os nossos serviços em um único lugar. O que isso significa?

Nosso sistema de pagamento atual deixa de ser o PagBank e passa a ser o Stripe,
Passamos a aceitar pagamentos em moedas estrangeiras
Renovação de pagamento de anuidade ou novos cadastros na associação devem ser realizadas no novo sistema (apyb.python.org.br/associados/associe-se). 

Lembrando que atualmente temos 2 modalidades para tornar-se Associada à APyB:

- Estudante: anuidade de R$ 32,00
- Efetivo: anuidade de R$ 64,00

Associando-se à APyB garante o uso da infraestrutura da APyB para organizar eventos, votar para alterações no Estatuto e candidatar-se para a Diretoria ou para o Conselho Deliberativo ou Conselho Fiscal.

Os valores arrecadados com as anuidades ajudam a manter a estrutura da APyB ativa.

Aproveite para conferir também a nova interface do nosso site oficial e da nossa documentação.

> O passo-a-passo de como acessar podes [acessar neste link](https://www.canva.com/design/DAGPAUji2p8/U_A3Hrkcwr0VzWuozGpp5g/view?utm_content=DAGPAUji2p8&utm_campaign=share_your_design&utm_medium=link&utm_source=shareyourdesignpanel#3)

Dúvidas? Envie um email para: [contato@python.org.br](mailto:contato@python.org.br)

<img src="https://github.com/user-attachments/assets/54703ca7-c7e3-45e0-9923-f25a4c01135d" width=250px alt="Imagem gráfica com o título Nova plataforma para associar-se a APyB e com o subtítulo Integração com Stripe" /> 

👋🏻 linktr.ee/apyb

**Comment by luzfcb** [comment]
Pergunta sincera: Foi feito contato oficial via pessoa jurídica da APyB com a Stripe afim de informa-los sobre o caso de uso da APyB e evitar problemas futuros?

Eu estou perguntando isso porquê Stripe recentemente foi responsável por bloquear valores de um desenvolvedor brasileiro, e deixa-lo a mingua até por fim a comunidade conseguiu chamar atenção para que ela desse atenção específica para o problema dele, então deixa-los cientes e ter prova documental de tudo talvez seja bom para evitar dor de cabeça futura para a APyB.

Não sei o quão relevante é o meu questionamento, mas resolvi fazê-lo para deixar registrada minhas preocupações.

**Reply by naanadr** [reply]
@luzfcb agradecemos a preocupação, e sim nosso cadastro na Stripe foi realizado como pessoa jurídica, com anexo de todos os documentos comprobatórios. Além disso, até o momento em que escrevo essa mensagem, continuamos recebendo mensalmente em nossa conta do BB, a remessa de dinheiro da Stripe. 

Espero ter respondido suas dúvidas, caso tenha mais algum questionamento, é só avisar 😄

**Reply by rennerocha** [reply]
@naanadr Por que foi feito como Pessoa Física? Isso pode se tornar um problema sério no futuro. O correto seria uma conta em nome da Pessoa Jurídica.

@luzfcb acho que se a APyB está retirando o dinheiro mensalmente para o Banco do Brasil, caso a Stripe crie algum problema, o risco diminui e o prejuízo financeira seria pequeno. Ideal mesmo é se fosse possível saques com maior regularidade (o PayPal por exemplo dá pra fazer saque automático semanal), mas acho que mensal já é uma boa segurança de que se eles criarem caso, não fiquemos com dinheiro preso.

**Reply by naanadr** [reply]
@rennerocha digitei errado ali na mensagem. O cadastro foi feito como **pessoa jurídica**, utilizando o estatuto da associação, assim como os dados bancários da mesma.

**Comment by rfloriano** [comment]
Pessoal, estava tentando ver meu status na página de membros da APyB e percebi que o link do `antigo portal de associados` não está levando para a antiga página. Parece haver algum redirect que manda pra home.

<img width="1728" height="1117" alt="Screenshot 2025-10-02 at 16 35 58" src="https://github.com/user-attachments/assets/de06b601-2441-47b3-b959-078b8bbd726d" />

https://apyb.python.org.br/associados/associe-se/#ja-e-associado

**Reply by naanadr** [reply]
Oi @rfloriano, boa noite. O antigo site dos associados já foi descontinuado, porque não temos mais nenhum associado com a membresia ativa nele. Dessa forma, pode seguir com a sua assinatura pelo Stripe 😄 ainda essa semana estaremos atualizando o site para remover as referências ao sistema antigo.

**Reply by rfloriano** [reply]
entendo. mas há algum lugar oficial público da APyB para ver a lista de membros ativos?
